### PR TITLE
storage account managed identity enabled

### PR DESCRIPTION
**Pull request by abdullahaslam306**

### CODE REVIEW

File: plugins/azure/mediaServices/amsStorageAccountIdentityEnabled.js [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+var helpers = require('../../../helpers/azure');
+
+module.exports = {
+    title: 'Media Services Storage Account Managed Identity',
+    category: 'Media Services',
+    domain: 'Content Delivery',
+    description: 'Ensure that Azure Media Services have managed identity enabled for Storage Account authentication.',
+    more_info: 'Enabling managed identity for storage authentication allows secure access to Azure Storage without explicit credentials, enhancing security and simplifying access management for Azure Media Services.',
+    link: 'https://learn.microsoft.com/en-us/azure/media-services/latest/concept-managed-identities#media-services-managed-identity-scenarios',
+    recommended_action: 'Modify the media service\'s storage account settings and enable managed identity.',
```
Comment (alphadev4) [inline plugins/azure/mediaServices/amsStorageAccountIdentityEnabled.js:11]:
```suggestion
    recommended_action: 'Modify Media Service storage account settings and enable managed identity.',
```

File: plugins/azure/mediaServices/amsStorageAccountIdentityEnabled.js [hunk 2]
```diff
@@ -0,0 +1,50 @@
+var async = require('async');
+var helpers = require('../../../helpers/azure');
+
+module.exports = {
+    title: 'Media Services Storage Account Managed Identity',
+    category: 'Media Services',
+    domain: 'Content Delivery',
+    description: 'Ensure that Azure Media Services have managed identity enabled for Storage Account authentication.',
```
Comment (alphadev4) [inline plugins/azure/mediaServices/amsStorageAccountIdentityEnabled.js:8]:
```suggestion
    description: 'Ensures that Azure Media Services have managed identity enabled for Storage Account authentication.',
```

**Review by alphadev4 — APPROVED** [review]

File: plugins/azure/mediaServices/amsStorageAccountIdentityEnabled.js [hunk 3]
```diff
@@ -0,0 +1,50 @@
+var async = require('async');
+var helpers = require('../../../helpers/azure');
+
+module.exports = {
+    title: 'Media Services Storage Account Managed Identity',
+    category: 'Media Services',
+    domain: 'Content Delivery',
+    description: 'Ensures that Azure Media Services have managed identity enabled for Storage Account authentication.',
```
Comment (alphadev4) [inline plugins/azure/mediaServices/amsStorageAccountIdentityEnabled.js:8]:
```suggestion
    description: 'Ensures that Azure Media Service accounts have managed identity enabled for Storage Account authentication.',
```

File: exports.js [hunk 4]
```diff
@@ -989,7 +989,9 @@ module.exports = {
         'eventHubMinimumTLSversion'     : require(__dirname + '/plugins/azure/eventhub/eventHubMinimumTLSversion.js'),
 
         'accessLogsEnabled'             : require(__dirname + '/plugins/azure/frontdoor/accessLogsEnabled.js'),
-        'frontDoorMinimumTlsVersion'    : require(__dirname + '/plugins/azure/frontdoor/frontDoorMinimumTlsVersion.js')
+        'frontDoorMinimumTlsVersion'    : require(__dirname + '/plugins/azure/frontdoor/frontDoorMinimumTlsVersion.js'),
+
+        'amsStorageAccountIdentityEnabled': require(__dirname + '/plugins/azure/mediaServices/amsStorageAccountIdentityEnabled.js')
```
Comment (mehakseedat63) [inline exports.js:994]:
```suggestion
        'amsStorageAccountIdentity': require(__dirname + '/plugins/azure/mediaServices/amsStorageAccountIdentityEnabled.js')
```
Filename could just be amsStorageAccountIdentity

**Review by mehakseedat63 — CHANGES_REQUESTED** [review]

**Pull request merged**